docs/development/contributing/ide.md

```diff
@@ -0,0 +1,32 @@
+# IDE configuration
+
+Using an integrated development environments (IDE) and configuring it properly will help you work on Polars more effectively.
+This page contains some recommendations for configuring popular IDEs.
+
+## Visual Studio Code
+
+The following extensions are recommended.
+
+### rust-analyzer
+
+If you work on the Rust code at all, you will need the [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) extension. This extension provides code completion for the Rust code.
+
+For it to work well for the Polars code base, add the following settings to your `.vscode/settings.json`:
+
+```json
+{
+    "rust-analyzer.cargo.features": "all",
+    "rust-analyzer.linkedProjects": ["py-polars/Cargo.toml"],
+}
+```
+
+### Ruff
+
+The Ruff extension will help you conform to the formatting requirements of the Python code.
+We use both the Ruff linter and formatter.
+
+## PyCharm / CLion
```

Comment:
...and RustRover :)
https://www.jetbrains.com/rust/

Reply:
Hadn't heard of this one yet! I'll add it, but in any case, further information for JetBrains IDEs will be added later after I better organize some sections.